_includes/v19.1/misc/basic-terms.md

```diff
@@ -2,7 +2,7 @@ Term | Definition
 -----|------------
 **Cluster** | Your CockroachDB deployment, which acts as a single logical application.
 **Node** | An individual machine running CockroachDB. Many nodes join together to create your cluster.
-**Range** | CockroachDB stores all user data (tables, indexes, etc.) and almost all system data in a giant sorted map of key-value pairs. This keyspace is divided into "ranges", contiguous chunks of the keyspace, so that every key can always be found in a single range.<br><br>From a SQL perspective, a table and its secondary indexes initially map to a single range, where each key-value pair in the range represents a single row in the table (also called the primary index because the table is sorted by the primary key) or a single row in a secondary index. As soon as a range reaches 64 MiB in size, it splits into two ranges. This process continues as the table and its indexes continue growing.
+**Range** | CockroachDB stores all user data (tables, indexes, etc.) and almost all system data in a giant sorted map of key-value pairs. This keyspace is divided into "ranges", contiguous chunks of the keyspace, so that every key can always be found in a single range.<br><br>From a SQL perspective, a table and its secondary indexes initially map to a single range, where each key-value pair in the range represents a single row in the table (also called the primary index because the table is sorted by the primary key) or a single row in a secondary index. As soon as that range reaches 64 MiB in size, it splits into two ranges. This process continues for the child ranges as the table and its indexes continue growing.
```

Comment:
I like that this is more explicit, but I'm a little concerned about the term `child`. There is no parent/child relationship, really, when it comes to ranges for table data. How about we just change `for the child ranges` to `for these ranges`?

Reply:
I agree with your assessment.